components/event-bus/cmd/event-bus-publish-knative/handlers/publish.go

```diff
@@ -20,6 +20,12 @@ var (
 	defaultChannelNamespace = knative.GetDefaultChannelNamespace()
 )
 
+type Message struct {
```

Comment:
Does it make sense to restrict the visibility of the **Message** struct to be on the package level? I do not think we need to access it from outside the package.

Reply:
Maybe in a later PR, since this one is merged.